manifests/u/Unknown-HorizonsTeam/Unknown-Horizons/2019.1/Unknown-HorizonsTeam.Unknown-Horizons.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Unknown-HorizonsTeam.Unknown-Horizons
+PackageVersion: 2019.1

```

```suggestion
PackageVersion: 2019.1
Scope: machine
```
Package installs machine-wide